src/sentry/tagstore/v2/models/grouptagkey.py

```diff
@@ -0,0 +1,88 @@
+"""
+sentry.tagstore.v2.models.grouptagkey
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2017 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+import six
+
+from django.db import router, transaction, DataError
+
+from sentry.api.serializers import Serializer, register
+from sentry.db.models import (
+    Model, BoundedPositiveIntegerField, BaseManager, sane_repr
+)
+
+
+class GroupTagKey(Model):
+    """
+    Stores a unique tag key name for a group.
+
+    An example key might be "url" or "server_name".
+    """
+    __core__ = False
+
+    project_id = BoundedPositiveIntegerField(db_index=True)
+    group_id = BoundedPositiveIntegerField(db_index=True)
+    environment_id = BoundedPositiveIntegerField()
+    key_id = BoundedPositiveIntegerField()
+    # values_seen will be in Redis
+
+    objects = BaseManager()
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_grouptagkey_v2'
```

Comment:
The only real criticism I have here now is the name of the tables.

I'd use an `app_label` if `tagstore`, so then we get table names such as `tagstore_grouptagkey`, etc. It'd be much nicer than `sentry_grouptagkey_v2` and will fall in line also with how we do `nodestore`.